components/ingest-service/grpc/grpc.go

```diff
@@ -54,12 +55,68 @@ func Spawn(opts *serveropts.Opts) error {
 		}).Error("could not connect to elasticsearch")
 		return err
 	}
+	grpcServer := opts.ConnFactory.NewServer()
+	uri := fmt.Sprintf("%s:%d", opts.Host, opts.Port)
 
-	var (
-		grpcServer = opts.ConnFactory.NewServer()
-		migrator   = migration.New(client)
-		uri        = fmt.Sprintf("%s:%d", opts.Host, opts.Port)
-	)
+	// Authz Interface
+	authzConn, err := opts.ConnFactory.Dial("authz-service", opts.AuthzAddress)
+	if err != nil {
+		// This should never happen
+		log.WithError(err).Error("Failed to create Authz connection")
+		return err
+	}
+	defer authzConn.Close() // nolint: errcheck
+
+	authzProjectsClient := authz.NewProjectsClient(authzConn)
+
+	// event feed Interface
+	eventFeedConn, err := opts.ConnFactory.Dial("event-feed-service", opts.EventFeedAddress)
+	if err != nil {
+		// This should never happen
+		log.WithError(err).Error("Failed to create Event Feed connection")
+		return err
+	}
+	defer eventFeedConn.Close() // nolint: errcheck
+
+	eventFeedServiceClient := event_feed.NewEventFeedServiceClient(eventFeedConn)
+
+	// Need to ensure the event feed service is ready for requests.
+	_, err = eventFeedServiceClient.GetFeed(context.Background(), &event_feed.FeedRequest{
```

Comment:
We were seeing problems with running the migration because the event-feed-service was returning errors because it was not fully started. This fails the service until the event-feed-service is ready to receive requests.